### DISCUSSION TITLE
Yes, it works with Android 14 (so far).

### DISCUSSION THREAD

**Discussion by arsradu** [kyujin-cho/pixel-volte-patch#27] [General]
Hi guys,

So, this is kind of an answer before the question kind of thing. :))

In case anyone was wondering if this patch still works with Android 14 (currently DP1), the answer is: yes. Yes, it does. 👍 

Great job, @kyujin-cho ! :)

**Comment by VlaDiRO88** [comment]
Good to hear, but lets wait for final release 👍
